obs/export.py

```diff
@@ -139,15 +139,10 @@ def checkForStandardKeysJSON():
     if 'checkinglevel' not in body_json.keys(): body_json['checkinglevel'] = args.checkinglevel
 
 def writeFile(outfile, p):
-    makeDir(outfile.rpartition('/')[0])
```

Comment:
@alerque Why remove this?  This making the parent directory if it doesn't exist.